.gitignore

```diff
@@ -25,6 +25,8 @@ venv/
 *.egg-info/
 .installed.cfg
 *.egg
+Pipfile
+Pipfile.lock
```

Comment:
Could you mention in the commit body why you added this?

Reply:
Sure.